config/default.yaml

```diff
@@ -89,3 +89,6 @@ unleash:
 
 metadata:
     community_attr: cyverse-community
+    instant_launch_location_attr: ui_location
+    instant_launch_dashboard: dashboard
+    instant_launch_nav_drawer: nav_drawer
```

Comment:
Am not entirely sure if these needs to be in config. I don't see them changing in different envs / external deployments.  They could live in some constants file and can be moved to configs when it is determined this will in fact be changing often. Also there is no real benefit to adding them in configs since we still have to redeploy the UI once the config changes ?

Reply:
Yeah, Paul made a similar comment about configs in another PR. It seems like we need to make a decision on how we want to handle them going forward.  It might be good to have @slr71 weigh in since she's been handling all the external deployments.

Reply:
I agree that these metadata attrs and values seem like they could be just integral to the DE system, and I'm not sure they would need to change between environments or deployments, like our system-use-only `ipc_*` iRODS metadata.

That said, maybe the metadata attributes could have a more specific name, in order to prevent any potential conflicts with future user defined metadata (like the `cyverse-community` attr above). So maybe `cyverse_de_ui_location`, for example?

In general I think configs are at least conceptually useful, and we should put values in configs that can (or should) change between environments or deployments, even if it requires a rebuild and redeploy for now. I just wish it were easier to change a config value in an environment and rolling-restart the ui service to pick up the new value, without having to push the change to a git repo and rebuild everything (especially if it's supposed to be just a temporary change, like disabling a feature for a little while).

Reply:
> That said, maybe the metadata attributes could have a more specific name, in order to prevent any potential conflicts with future user defined metadata (like the `cyverse-community` attr above). So maybe `cyverse_de_ui_location`, for example?
> 

Oh, I just noticed `ui_location` was already in use by older code, so scratch that suggestion 😅

Reply:
Hmm, I’m not sure I see a reason to make that configurable. I suppose that language differences might mean that users from other countries would want to user terms that make sense to them, but probably only if they had to set the attributes directly. Since we’re planning to build a UI for it, I would think we could use constants instead.

Reply:
Cool - I'll refactor to use constants then.  Thanks for the input everybody!